app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -23,11 +23,30 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
     val message = SingleLiveEvent<String>()
     val event = MutableLiveData<Event>()
     var attendee = MutableLiveData<User>()
+    var paymentSelectorVisibility = MutableLiveData<Boolean>()
 
     fun getId() = authHolder.getId()
 
     fun isLoggedIn() = authHolder.isLoggedIn()
 
+    fun updatePaymentSelectorVisibility(ticketIdAndQty: List<Pair<Int, Int>>?) {
+        val ticketIds = ArrayList<Int>()
+        ticketIdAndQty?.forEach { if (it.second > 0) ticketIds.add(it.first) }
+
+        compositeDisposable.add(ticketService.getTicketPriceWithIds(ticketIds)
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe({
+                    var total = 0.toFloat()
+                    it?.forEach {
+                        if (it.toFloat() > 0) total += it.toFloat()
+                    }
+                    paymentSelectorVisibility.value = total != 0.toFloat()
```

Comment:
Why this toFloat, toFloat everywhere

Reply:
because the price is a string in ticket model I am creating an issue about it,( since this is out of scope of this pr)

Reply:
@iamareebjamal i have created issue #290 for this